src/modules/remote.json

```diff
@@ -0,0 +1,157 @@
+        "summary": "The state of currently paired remote",
+        "schema": {
+          "$ref": "#/components/schemas/RemotePairState"
+        }
+      },
+      "examples": [
+        {
+          "name": "Remote State Examples",
+          "params": [],
+          "result": {
+            "pairingState": "pairingRemote"
+          }
+        }
+      ]
+    },
+    {
+      "name": "onRemotePairing",
+      "tags": [
+        {
+          "name": "event"
+        }
+      ],
+      "summary": "Listen for events when the remote control is pairing.",
+      "params": [
+        {
+          "name": "listen",
+          "required": true,
+          "summary": "whether to listen, or stop listening",
+          "schema": {
+            "type": "boolean"
+          }
+        }
+      ],
+      "result": {
+        "name": "remotePairState",
+        "summary": "The state of currently paired remote",
+        "schema": {
+          "schema": {
+            "oneOf": [
+              {
+                "$ref": "https://meta.comcast.com/firebolt/types#/definitions/ListenResponse"
+              },
+              {
+                "$ref": "#/components/schemas/RemotePairState"
+              }
+            ]
+          }
+        }
+      },
+      "examples": [
+        {
+          "name": "Default Example",
+          "params": [
+            {
+              "name": "listen",
+              "value": true
+            }
+          ],
+          "result": {
+            "name": "event",
+            "value": {
+              "remotePairState": "pairingComplete"
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "components": {
+    "schemas": {
+      "RemoteControlState": {
+        "title": "RemoteList",
+        "type": "array",
+        "description": "Contains a list of remotes paired to the device.",
+        "items": {
+          "$ref": "#/components/schema/RemoteState"
+        }
+      },
+      "RemoteState": {
+        "title": "RemoteState",
+        "type": "object",
+        "description": "The type of keyboard to show to the user",
+        "minProperties": 2,
+        "additionalProperties": false,
+        "required": ["model", "batteryPercentage"],
+        "properties": {
+          "model": {
+            "type": "string",
+            "description": "Model of the remote."
+          },
+          "batteryPercentage": {
+            "description": "Battery percentage value of the selected remote.",
+            "type": "integer",
+            "default": 0
+          }
+        }
+      },
+      "RemotePairState": {
+        "title": "RemotePairState",
+        "type": "object",
+        "description": "Contains the status for the current remote pairing state",
+        "minProperties": 1,
+        "additionalProperties": false,
+        "required": ["pairingState"],
+        "properties": {
+          "pairingState": "string",
+          "enum": [
+            "pairingRemote",
+            "waitingForKeypress",
```

Comment:
what is the difference between pairingRemote and waitingForKeypress?

Reply:
Updated the code to remove waiting for keypress seems specific to the underlying platform